File: src/client/views/heading.ts
// AUTO GENERATED @ 2024-05-17T16:46:47.978Z
export interface HeadingProps {
  content: string;
}

export function heading(props: HeadingProps) {
  return `<h1 class="p-2 font-bold text-xl">${props.content}</h1>`;
}

File: src/server/views/heading.rs
// AUTO GENERATED @ 2024-05-17T16:46:47.982Z
pub struct HeadingProps {
    pub content: String,
}

pub fn heading(props: HeadingProps) -> String {
    let content = props.content;

    return format!(r#"<h1 class="p-2 font-bold text-xl">{content}</h1>"#);
}